lib/jekyll/readers/data_reader.rb

```diff
@@ -56,6 +56,12 @@ def read_data_file(path)
           :headers  => true,
           :encoding => site.config["encoding"],
         }).map(&:to_hash)
+      when ".tsv"
+        CSV.read(path, {
+          :col_sep => "\t",
```

Comment:
@ashmaroli pointed out that rubocop correctly flagged this as incorrectly spaced. Run `script/fmt -a` to auto-correct it.